src/sensors.c

```diff
@@ -155,7 +160,9 @@ void init_sensors(void)
   mpu6050_register_interrupt_cb(&imu_ISR);
 
   uint16_t acc1G;
-  mpu6050_init(true, &acc1G, &gyro_scale, _params.values[PARAM_BOARD_REVISION]);
+  float gyro_scale_to_urad;
+  mpu6050_init(true, &acc1G, &gyro_scale_to_urad, _params.values[PARAM_BOARD_REVISION]);
+  gyro_scale = 1000000.0*gyro_scale_to_urad;
```

Comment:
The problem is that first equation.  I'll put it all in scientific notation so it makes sense

`x e-6 rad * (1 rad / 1e6) = x e-12 rad` not `x rad` .

It should be `x e-6 rad * (1e6) = x rad`